platform/iphone/display_server_iphone.mm

```diff
@@ -54,6 +54,8 @@
 #if defined(GLES3_ENABLED)
 	// FIXME: Add support for both OpenGL and Vulkan when OpenGL is implemented
 	// again,
+	// Note that we should be checking "opengl3" as the driver, might never enable this seeing OpenGL is deprecated on iOS
```

Comment:
As mentioned in the OP, I added these comments to clear up why we're checking `opengl_es` and point to the code later on that forces the vulkan driver to be active.